setup/monitoring/index.md

```diff
@@ -31,6 +31,13 @@ You can also use the microservice HTTP endpoint `/spectator/metrics`
 directly to scrape metrics yourself. The JSON document structure is
 further documented in the Monitoring reference section.
 
+Note that there are numerous instrumented places within the codebase, not all
+of which are of interest to operators or non-mega-scale users (e.g. for tuning
+large scale performance issues as the envelope is pushed further). The
```

Comment:
I think we might be able to get away with deleting the first sentence, and just get straight to "The daemon can be configured..."